docs/contributing/testing/database/index.md

```diff
@@ -0,0 +1,82 @@
+# Database Integration Testing
+
+Since Bitwarden has multiple database options, testing them all automatically is incredibly
+important so that we don't have to have a full QA process for each database type. This is where the
+`Infrastructure.IntegrationTest` project comes in, allowing the setup of tests similarly to how the
+databases are consumed in the applications through their common interfaces, generally an
+`I*Repository`. These tests are automatically executed on all supported databases in our testing
+pipeline.
+
+## Creating a new test
+
+To create a new database test just add the `[DatabaseTheory]` and `[DatabaseData]` attributes to
+test. Now, in the parameters of the test you are able to "inject" any repository layer services
+directly into the test. The test will run for every database that is
+[configured in the current environment](#configure-the-tests). Since you inject the interface of the
+service, some runs of the test will use the Dapper based repository implementation targeting
+Microsoft SQL Server and others will use the Entity Framework Core based implementations, which we
+use for MySql, Postgres, and SQLite.
+
+The goal of database tests is to test the business logic that is encapsulated in a given method. If
+the stored procedure in SQL Server then calls another procedure to update the
+`User.AccountRevisionDate` then the same EF implementation should do that as well. By running the
+test against all variants we are ensuring all the variants are feature-equal. Locally, you may want
+to only run the SQL Server tests along with one EF implementation; SQLite is often the easiest in
+that situation. This may work well for a very long time and save you some time overall but there are
+differences between the EF database providers such that you will one day get errors in the CI
+pipeline.
+
+## Configure the databases
+
+The databases are expected to have the latest migrations applied.
```

Comment:
Link to your instructions on this?

Reply:
They are already linked below for running the tests locally. I don't really think this section is offering anything and opted to remove it instead.